content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -35,7 +35,7 @@ secretGenerator:
   - password=1f2d1e2e67df
 {{< /tab >}}
 {{% tab name="Files" %}}
-1.  Store the credentials in files with the values encoded in base64:
+1.  Store the credentials in files. The filenames will be the keys of the secret:
```

Comment:
Hi @Henni .
In the example, since it appears that the values are not base64 encoded, you would like to remove that text?

Later in this page there is a supporting statement: `In all cases, you don't need to base64 encode the values.`

Possible edit: The filenames are the keys of the secret.

Reply:
> In the example, since it appears that the values are not base64 encoded, you would like to remove that text?

Yes this is the intent of this PR.

> Possible edit: The filenames are the keys of the secret.

👍 applied